src/no-important.js

```diff
@@ -1,4 +1,19 @@
-import aphrodite from 'aphrodite/no-important';
+import { StyleSheet, css } from 'aphrodite/no-important';
 import aphroditeInterfaceFactory from './aphroditeInterfaceFactory';
 
-export default aphroditeInterfaceFactory(aphrodite);
+let flushToStyleTag;
+try {
+  // Aphrodite 1
+  // eslint-disable-next-line import/no-unresolved, global-require, prefer-destructuring
+  flushToStyleTag = require('aphrodite/lib/inject').flushToStyleTag;
```

Comment:
this kind of still seems like a breaking change; i wonder if we should put this try/catch in a separate file and require that wherever we'd otherwise have required `aphrodite/lib/inject`?

Reply:
Sorry I don't follow. What about this is breaking?

Reply:
anyone who's invoking the factory function via deep-import

Reply:
ah I see